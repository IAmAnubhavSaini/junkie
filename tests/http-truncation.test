#!/bin/sh

NAME="HTTP-Truncation"
PCAPS="pcap/truncation/http-96.pcap"
filter() {
	grep '^HTTP' | cut -d' ' -f 2-
}
OUTPUT="http-truncation.out"

. "$srcdir/dump_test"
