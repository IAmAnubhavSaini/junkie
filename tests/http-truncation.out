-- pcap/truncation/http-96.pcap --
head_len=24, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=le, url=/
head_len=24, payload=0, method=unset, code=301, content_length=unset, transfert_encoding=unset, mime_type=unset, host=unset, url=unset
head_len=24, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=unset, url=/content/widgets/exp
head_len=24, payload=0, method=unset, code=301, content_length=unset, transfert_encoding=unset, mime_type=unset, host=unset, url=unset
head_len=24, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=unset, url=/com/bazarchic-box/m
head_len=24, payload=0, method=unset, code=301, content_length=unset, transfert_encoding=unset, mime_type=unset, host=unset, url=unset
